src/framework/preferences/js/Panels.js

```diff
@@ -841,7 +841,7 @@ var fluid_2_0 = fluid_2_0 || {};
             theme: ["default", "bw", "wb", "by", "yb", "lgdg"]
         },
         markup: {
-            label: "<span class=\"fl-preview-A\">A</span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
+            label: "<span class=\"fl-preview-A\"></span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
```

Comment:
While the "a" is no longer in the markup. It seems that the screen reader is still picking it up. This is probably because the "a" content added via css is still picked up by the screen reader as part of the label.